iml-manager-cli/src/stratagem.rs

```diff
@@ -323,21 +332,65 @@ pub async fn stratagem_cli(command: StratagemCommand) -> Result<(), ImlManagerCl
 
             wait_for_cmd_display(command).await?;
         }
-        StratagemCommand::Filesync(data) => {
-            let query = stratagem_queries::filesync::build(
-                &data.filesystem,
-                data.remote,
-                data.expression,
-                "push",
-            );
-
-            let resp: iml_graphql_queries::Response<stratagem_queries::filesync::Resp> =
-                graphql(query).await?;
+        StratagemCommand::Filesync(data) => match data.expression {
+            Some(ref _x) => {
+                let query = stratagem_queries::filesync::build(
+                    &data.filesystem,
+                    data.remote,
+                    data.expression.unwrap(),
```

Comment:
Avoid usage of `.unwrap()`

Reply:
I can't here.  I've verified that data.expression is valid, and I need it's contents, which is expressly what unwrap is for.

Reply:
If a function returns a `Result` (which this one does) you can always return an `Err` instead.